robocop/checkers/errors.py

```diff
@@ -90,3 +92,18 @@ def visit_KeywordCall(self, node):  # noqa
                 lineno=node.lineno,
                 col=node.data_tokens[0].col_offset + 1
             )
+
+
+class VariablesImportErrorChecker(VisitorChecker):
+    """ Checker for syntax error in variables import. """
+    rules = {
+        "0404": (
+            "variables-import-with-args",
```

Comment:
I am thinking if we should use shortcuts in rule names. What is better - `"variables-import-with-args"` or `"variables-import-with-arguments"`?

Reply:
We can go with longer one - I've used shortened form for people who actually type the names instead of ids (--include ...)

Reply:
I think you're right. For name it can be shorter, because people need to actually write it. On the other hand, I think this is another thing we did not standardize and probably we have rules with 'args' and 'arguments', 'vars' and 'variables'...

Reply:
Right, I will save if for review later